integrations/cross_platform/yocto.rst

```diff
@@ -0,0 +1,369 @@
+
+Three stages can be differentiated in the proposed flow:
+
+1. Developers can create an application with the native tools in their desktop platform of choice using their usual IDE, compiler or
+debugger and test the application.
+
+   .. image:: /images/yocto/conan-yocto_development.png
+       :height: 600 px
+       :width: 600 px
+       :align: center
+
+2. Packages can be cross-built for the target device using the Yocto SDK and uploaded to Artifactory, even automated in a CI process.
+
+   .. image:: /images/yocto/conan-yocto_cross-build.png
+       :height: 600 px
+       :width: 600 px
+       :align: center
+
+3. Once the cross-built packages are available in Artifactory, the application can be directly deployed to the Yocto image. This step can also be automated also in a CI.
+   it from sources again.
+
+   .. image:: /images/yocto/conan-yocto_deploy.png
+       :height: 450 px
+       :width: 450 px
+       :align: center
+
+Creating Conan packages with Yocto's SDK
+========================================
+
+Prepare your recipes
+--------------------
+
+First of all, the recipe of the application to be deployed to the final image should have a
+`deploy() method <https://docs.conan.io/en/latest/devtools/running_packages.html>`_. There you can specify the files of the application
+needed in the image as well as any other from its dependencies (like shared libraries or assets):
+
+.. code-block:: python
+   :caption: *conanfile.py*
+   :emphasize-lines: 28-31
+
+    from conans import ConanFile
+
+
+    class MosquittoConan(ConanFile):
+        name = "mosquitto"
+        version = "1.4.15"
+        description = "Open source message broker that implements the MQTT protocol"
+        license = "EPL", "EDL"
+        settings = "os", "arch", "compiler", "build_type"
+        generators = "cmake"
+        requires = "OpenSSL/1.0.2o@conan/stable", "c-ares/1.14.0@conan/stable"
+
+    def source(self):
+        source_url = "https://github.com/eclipse/mosquitto"
+        tools.get("{0}/archive/v{1}.tar.gz".format(source_url, self.version))
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        self.copy("*.h", dst="include", src="hello")
+        self.copy("*.so", dst="lib", keep_path=False)
+        self.copy("*.a", dst="lib", keep_path=False)
+        self.copy("*mosquitto.conf", dst="bin", keep_path=False)
+
+    def deploy(self):
+        # Deploy the executables from this eclipse/mosquitto package
+        self.copy("*", src="bin", dst="bin")
+        # Deploy the shared libs from this eclipse/mosquitto package
+        self.copy("*.so*", src="lib", dst="bin")
+       # Deploy all the shared libs from the transitive deps
+        self.copy_deps("*.so*", src="lib", dst="bin")
+
+    def package_info(self):
+        self.cpp_info.libs.extend(["libmosquitto", "rt", "pthread", "dl"])
```

Comment:
```python
self.cpp_info.libs = ["mosquito", "rt", ....]
```

It is more clear that those are the libraries, and there are no other previous libraries in that list.
Furthermore, is it necessary to define ``libmosquitto``, or ``mosquito`` would work? Most libraries work without the ``lib`` prefix in Linux.